src/core/extendRoutes.spec.ts

```diff
@@ -251,4 +251,20 @@ describe('EditableTreeNode', () => {
       },
     ])
   })
+
+  it('can override the path', () => {
+    const tree = new PrefixTree(RESOLVED_OPTIONS)
+    const editable = new EditableTreeNode(tree)
+    const parent = editable.insert('parent', 'file.vue')
+    const child = parent.insert('child', 'file.vue')
+
+    child.path = 'relative'
+    parent.path = 'relative'
+    expect(parent.path).toBe('/parent')
```

Comment:
shouldn't this be `relative`

Reply:
That was intentional to test the warnings. Is there a method you use to test warnings It should log this warning:

> Only absolute paths are supported at the root of the route tree. Make sure that "relative" starts with a slash "/".`

The idea is that you would only be restricted to absolute paths at the root and the root's children.